app/assets/javascripts/govuk_publishing_components/lib/govspeak/youtube-link-enhancement.js

```diff
@@ -12,13 +12,41 @@
     if (!this.campaignCookiesAllowed()) {
       this.start = this.startModule.bind(this)
       window.addEventListener('cookie-consent', this.start)
+      this.decorateLink()
       return
     }
     this.startModule()
   }
 
+  YoutubeLinkEnhancement.prototype.decorateLink = function () {
+    var placeholder = document.createElement('p')
+    placeholder.setAttribute('class', 'gem-c-govspeak__youtube-placeholder govuk-body')
+
+    var $youtubeLinks = this.$element.querySelectorAll('a[href*="youtube.com"], a[href*="youtu.be"]')
+    for (var i = 0; i < $youtubeLinks.length; ++i) {
+      var $link = $youtubeLinks[i]
```

Comment:
Does the link text still appear somewhere? If not, it might be hard for users to work out what the video is about without it.

Reply:
Good shout, have updated (see new screenshot in description). Also in the process of changing this I found and fixed a bug, so all good 👍

Reply:
Nice one!